apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -318,9 +319,11 @@ export default function ListBoxInline({ app, fieldIdentifier, stateName = '$', o
           </Grid>
         </Grid>
       )}
-      <Grid item ref={searchContainerRef}>
-        <ListBoxSearch model={model} dense={dense} keyboard={keyboard} visible={searchVisible} />
-      </Grid>
+      {!disableSearch && (
```

Comment:
Why adding another condition here? Would it not be possible to use the already existing searchVisible? Just add the disableSearch check there?

Reply:
if we forcefully pass `visible={false}` to `ListBoxSearch` component, we don't even need to modify `searchVisible` in order to get our desired results || adding another prop to the component. 
Since the `<ListBoxSearch />` already checks for:
```tsx
if (!visible) {
    return null;
  }
``` 
inside of itself, then i'm pretty sure that should do the work.

the only remaining thing is the search icon which comes from sense-client. it still re-renders the react part when clicking on it

![Screenshot 2022-07-12 at 09 55 38](https://user-images.githubusercontent.com/29652890/178439131-c7e5cc94-b507-43fd-a358-b8f8525292c9.png)

I think we will need another PR for it in sense client in order to hide it temporarily

Reply:
I think `disableSearch` and `searchVisible` refer to different things, that's why the prop was added and not re-used

Reply:
because the listbox has a toggle-able searchfield.